torch/_dynamo/variables/functions.py

```diff
@@ -309,15 +309,19 @@ def call_function(
         # Note this is not only for allowed modules, as user customized modules can extend from
         # allowed modules but using parent's `forward` method, which is also covered by this branch.
         if isinstance(self.obj, variables.NNModuleVariable):
-            module_attr = getattr(self.fn, "__module__", "")
-            if (
-                module_attr is not None
-                and module_attr.startswith("torch.nn.")
-                or self.is_constant
-            ):
-                return self.obj.call_method(
-                    tx, self.fn.__name__, args, kwargs, constant=self.is_constant
-                ).add_options(self)
+            try:
+                return super().call_function(tx, args, kwargs)
+            except Unsupported:
+                module_attr = getattr(self.fn, "__module__", "")
+                if (
+                    module_attr is not None
+                    and module_attr.startswith("torch.nn.")
+                    or self.is_constant
+                ):
+                    return self.obj.call_method(
+                        tx, self.fn.__name__, args, kwargs, constant=self.is_constant
+                    ).add_options(self)
+
```

Comment:
the logic can probably be cleaned up here

Reply:
this might be wrong - we may be adding a try-catch to a path that was already going to be inlined, leading to fewer graph breaks than expected.